lib/yearRange.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+/* eslint-disable */
+
+/**
+ * Function to generate a span of years based on minimum and maximum parameters
+ *
+ * @param {Object} input - inputs in this plugin's instance and their values as {String}
+ * @param {Object} minYear - inputs minimum year value {String}
+ * @param {Object} maxYear - inputs maxmium year value {String}
+ * *
+ * @module yearRange
+ */
+
+module.exports = function yearRange(input, minYear, maxYear) {
+  var select = input.querySelector('.year-range');
+
+  for (var i = minYear; i <= maxYear; i++) {
```

Comment:
Just found out that nunjucks has a range setting, which I was unaware of. This seems perfect for our use case. I'm wondering if we add the range min and max to settings, if we can put nested values inside the nunjucks range function like 

```
{% for i in range({{settings.min}}, {{settings.max}}) -%}
  <option value="{{ i }}">{{ i }}</option>,
{%- endfor %}
```

 You can see it here:

https://mozilla.github.io/nunjucks/templating.html#range-start-stop-step